daemon/ZilliqaDaemon_AWS.cpp

```diff
@@ -73,6 +74,11 @@ enum SyncType : unsigned int {
   DB_VERIF
 };
 
+string currentTimeStamp() {
+  auto t = chrono::system_clock::to_time_t(chrono::system_clock::now());
+  return "[" + string(ctime(&t)) + "] : ";
```

Comment:
Now the logs look like this:
```
[Thu Jul  4 03:09:55 2019
] : Process zilliqa exists in 1 instances
```
Can you help to fix the abnormal newline before `]`?

Reply:
Fixed